roles/backup/templates/management-pod.yml.j2

```diff
@@ -24,6 +24,10 @@ spec:
     resources:
       {{ backup_resource_requirements | to_nice_yaml(indent=2) | indent(width=6, first=False) }}
 {%- endif %}
+{% if db_management_pod_node_selector %}
+      nodeSelector:
```

Comment:
```suggestion
  nodeSelector:
```

Reply:
nodeSelector is at the same level as containers, the indention here looks incorrect